### PR TITLE
add start_swarm_manager use case

### PR DESCRIPTION
**Pull request by Bonczidai**
Signed-off-by: Bonczidai Levente <levente.bonczidai@hp.com>

### CODE REVIEW

File: test/io/cloudslang/docker/swarm/start_swarm_manager.inputs.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#   (c) Copyright 2014 Hewlett-Packard Development Company, L.P.
+#   All rights reserved. This program and the accompanying materials
+#   are made available under the terms of the Apache License v2.0 which accompany this distribution.
+#
+#   The Apache License is available at
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+########################################################################################################
+test_start_swarm_manager_success:
+  testSuites: [swarm_local]
```
Comment (meirwah) [inline test/io/cloudslang/docker/swarm/start_swarm_manager.inputs.yaml]: why can't you run swarm on travis/circleci?

Comment (Bonczidai) [inline test/io/cloudslang/docker/swarm/start_swarm_manager.inputs.yaml]: well, we need Docker hosts for it - maybe we can use multiple dind containers

Comment (meirwah) [inline test/io/cloudslang/docker/swarm/start_swarm_manager.inputs.yaml]: that's one possibility...
but swarm_manager can not run on a single host?

Comment (meirwah) [inline test/io/cloudslang/docker/swarm/start_swarm_manager.inputs.yaml]: @Bonczidai ?

Comment (Bonczidai) [inline test/io/cloudslang/docker/swarm/start_swarm_manager.inputs.yaml]: working on a local test first, then will try to align it with travis / circleci

**Comment by Bonczidai** [comment]
migrated to #226

**Comment by orius123** [comment]
@Bonczidai can we delete the branch?

**Comment by Bonczidai** [comment]
yes - I am deleting the branch

**Pull request closed without merging**